docs/pages/world/systems.mdx

```diff
@@ -54,6 +54,61 @@ There are two ways to call one `System` from another one.
 | `_msgValue()`                                | zero                                                                                                                                            | can use [`WorldContextProvider`](https://github.com/latticexyz/mud/blob/main/packages/world/src/WorldContext.sol#L180-L203) to transfer the correct information |
 | Can be used by systems in the root namespace | No (it's a security measure)                                                                                                                    | Yes                                                                                                                                                             |
 
+#### Calling from a root `System`
+
+If you need to call a `System` from a `System` in the root namespace you can use [`SystemSwitch`](https://github.com/latticexyz/mud/blob/main/packages/world-modules/src/utils/SystemSwitch.sol).
+
+1. Import `SystemSwitch`.
+
+   ```solidity copy
+   import { SystemSwitch } from "@latticexyz/world-modules/src/utils/SystemSwitch.sol";
+   ```
+
+1. Call the function using `SystemSwitch.call`.
+   For example, here is how you can call [`IncrementSystem.increment()`](http://localhost:3000/templates/typescript/contracts#incrementsystemsol).
+
+   ```solidity copy
+    uint32 retval = abi.decode(
+      SystemSwitch.call(
+        abi.encodeWithSignature("increment()")
```

Comment:
I think we should recommend usage of `abi.encodeCall` over `abi.encodeWithSignature` because it gives better type safety. The "downside" is it requires importing the system/interface to call (or to define an interface locally).